js/components/graph/Node.js

```diff
@@ -124,19 +112,15 @@ export default class Node extends React.Component {
 }
 
 Node.propTypes = {
-  childs: PropTypes.array,
   className: PropTypes.string,
   editMode: PropTypes.bool,
   highlighted: PropTypes.bool,
   hybrid: PropTypes.bool,
-  inEdges: PropTypes.array,
   JSON: PropTypes.object,
   onClick: PropTypes.func,
   onMouseEnter: PropTypes.func,
   onMouseLeave: PropTypes.func,
-  outEdges: PropTypes.array,
   status: PropTypes.string,
-  selected: PropTypes.bool,
   parents: PropTypes.array,
   svg: PropTypes.object,
```

Comment:
You can remove this prop as well, and instead pass in the specific callback functions that are needed (`onWheel`, `onKeyDown`)